app/experimenter/templates/experiments/detail_review.html

```diff
@@ -37,8 +39,16 @@ <h5>Sign-Off Checklist</h5>
               <label>
                 {{ field }}
                 {{ field.label }}
-                {{ field.help_text|safe }}
               </label>
+              {% if field.name == 'review_intent_to_ship' %}
+                <button type="button" class="btn btn-info send-intent-to-ship"
```

Comment:
I think we can check the review flag here, and if it's false only show the button in this spot, and if it's true, only show the checkbox.